lib/preboots/nsq.js

```diff
@@ -0,0 +1,41 @@
+const nsq = require('nsq.js-k8');
+
+module.exports = function nsqboot(app, options, callback) {
+  //
+  // So we will need a good way to get the IP addresses for the nslookupd
+  //
+  const config = app.config.get('nsq');
+  if (!config) return callback();
+  let finished = false;
+  //
+  // NSQLOOKUPD doesnt quite get it right when fetching hosts.
+  // We manually add the full DNS extension so the given hostname works in
+  // every namespace.
+  //
+  config.addrModify = (addr) => {
```

Comment:
does this function need to exist in the context of `config` or would a regular function suffice here?